base_local_planner/include/base_local_planner/goal_functions.h

```diff
@@ -117,6 +118,7 @@ namespace base_local_planner {
   bool getGoalPose(const tf::TransformListener& tf,
   		  const std::vector<geometry_msgs::PoseStamped>& global_plan,
   		  const std::string& global_frame,
+        const ros::Time& time,
```

Comment:
The indentation here is off -- but it appears that is because you properly used spaces, but the lines above/below use some tabs.

Reply:
Will fix.